libioc/Provisioning/__init__.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         self,
         value: _SourceInputType
     ) -> None:
-       self.value = value
+        self._value = value
```

Comment:
`self.value` would process the input. The plain storage property should not be written here.

Reply:
aye aye